### ISSUE TITLE
appveyor is failing on pre-install

### ISSUE THREAD

**Issue by tacaswell** [matplotlib/matplotlib#6657]
```
set LIBRARY_LIB=%CONDA_DEFAULT_ENV%\Library\lib
mkdir lib || cmd /c "exit /b 0"
A subdirectory or file lib already exists.
copy %LIBRARY_LIB%\zlibstatic.lib lib\z.lib
The system cannot find the path specified.
Command exited with code 1
```

**Comment by janschulz** [comment]
This should be fixed by the `CONDA_DEFAULT_ENV` -> `CONDA_PREFIX` change?

**Comment by tacaswell** [comment]
Closed by #6676 and #6520
